config/formats.js

```diff
@@ -562,6 +562,36 @@ let Formats = [
 		ruleset: ['Pokemon', 'Standard NEXT', 'Team Preview'],
 		banlist: ['Uber'],
 	},
+	{
+		name: '[Gen 7] Shitmons',
+		threads: [
+			`&bullet; <a href="http://spo.ink/shitmons">Shitmons</a>`,
+		],
+
+		gameType: 'doubles',
+		teamLength: {
+			validate: [2, 2],
+			battle: 2,
+		},
+		ruleset: ['Cancel Mod', 'HP Percentage Mod'],
+		banlist: [
+			'Honedge', 'Doublade', 'Aegislash', 'Kitsunoh', 'Shedinja ++ Sturdy', 'Power Construct', 'Schooling', 'Iron Barbs', 'Rough Skin',
+			'Pure Power', 'Huge Power', 'Cursed Body', 'Moody', 'Pressure', 'Wonder Guard', 'Fur Coat', 'Parental Bond', 'Protean', 'Stamina',
+			'Snow Warning', 'Sand Stream', 'Primordial Sea', 'Desolate Land', 'Poison Heal', 'Ice Body', 'Rain Dish', 'Grassy Surge',
+			'Cheek Pouch', 'Aguav Berry', 'Berry', 'Berry Juice', 'Black Sludge', 'Enigma Berry', 'Figy Berry', 'Iapapa Berry', 'Leftovers',
+			'Mago Berry', 'Oran Berry', 'Sitrus Berry', 'Wiki Berry', 'Rocky Helmet', 'Normalium-Z', 'Aerodactylite', 'Aggronite', 'Ampharosite',
+			'Blastoisinite', 'Blazikenite', 'Charizardite X', 'Charizardite Y', 'Diancite', 'Galladite', 'Garchompite', 'Gardevoirite',
+			'Gyaradosite', 'Latiasite', 'Latiosite', 'Lucarionite', 'Metagrossite', 'Salamencite', 'Sceptilite', 'Steelixite', 'Swampertite',
+			'Tyranitarite', 'Venusaurite',
+		],
+		onValidateSet: function (set) {
+			let template = this.getTemplate(set.species);
+			let bst = template.baseStats.hp + template.baseStats.atk + template.baseStats.def + template.baseStats.spa + template.baseStats.spd + template.baseStats.spe;
+			if (set.name === template.baseSpecies) return [`${set.species} does not have a nickname set.`, `(Edgy nicknames are required in Shitmons)`];
+			if (bst > 600) return [`${template.species} has a too high BST for Shitmons.`, `(You are limited to Pok\u00e9mon with 600 BST or lower)`];
```

Comment:
it would make sense to have this called after the `bst` variable is handled, and i feel like the wording is better as `${template.name} (${template.species}) has a BST too high for Shitmons.`, and the second part sounds better as `(Pok\u00e9mon with a BST higher than 600 are banned)` imo (also change`return [];` to `problems.push();`)